assets/layers/bike_parking/bike_parking.json

```diff
@@ -73,68 +73,68 @@
           }
         },
         {
           "if": "bicycle_parking=rack",
           "then": {
-            "en": "Rack <img width='150px' src='./assets/layers/bike_parking/rack.svg'>",
-            "nl": "Rek <img width='150px' src='./assets/layers/bike_parking/rack.svg'>",
-            "fr": "Râtelier <img width='150px' src='./assets/layers/bike_parking/rack.svg'>",
-            "gl": "Cremalleira <img width='150px' src='./assets/layers/bike_parking/rack.svg'>",
-            "de": "Gestell <img width='150px' src='./assets/layers/bike_parking/rack.svg'>"
+            "en": "Rack <img style='width: 25%'' src='./assets/layers/bike_parking/rack.svg'>",
+            "nl": "Rek <img style='width: 25%'' src='./assets/layers/bike_parking/rack.svg'>",
+            "fr": "Râtelier <img style='width: 25%'' src='./assets/layers/bike_parking/rack.svg'>",
+            "gl": "Cremalleira <img style='width: 25%'' src='./assets/layers/bike_parking/rack.svg'>",
+            "de": "Gestell <img style='width: 25%'' src='./assets/layers/bike_parking/rack.svg'>"
           }
         },
         {
           "if": "bicycle_parking=two_tier",
           "then": {
-            "en": "Two-tiered <img width='150px' src='./assets/layers/bike_parking/two_tier.svg'>",
-            "nl": "Dubbel (twee verdiepingen) <img width='150px' src='./assets/layers/bike_parking/two_tier.svg'>",
-            "fr": "Superposé <img width='150px' src='./assets/layers/bike_parking/two_tier.svg'>",
-            "gl": "Dobre cremalleira <img width='150px' src='./assets/layers/bike_parking/two_tier.svg'>",
-            "de": "Zweistufig <img width='150px' src='./assets/layers/bike_parking/two_tier.svg'>"
+            "en": "Two-tiered <img style='width: 25%'' src='./assets/layers/bike_parking/two_tier.svg'>",
+            "nl": "Dubbel (twee verdiepingen) <img style='width: 25%'' src='./assets/layers/bike_parking/two_tier.svg'>",
+            "fr": "Superposé <img style='width: 25%'' src='./assets/layers/bike_parking/two_tier.svg'>",
+            "gl": "Dobre cremalleira <img style='width: 25%'' src='./assets/layers/bike_parking/two_tier.svg'>",
+            "de": "Zweistufig <img style='width: 25%'' src='./assets/layers/bike_parking/two_tier.svg'>"
           }
         },
         {
           "if": "bicycle_parking=shed",
           "then": {
-            "en": "Shed <img width='150px' src='./assets/layers/bike_parking/shed.svg'>",
-            "nl": "Schuur <img width='150px' src='./assets/layers/bike_parking/shed.svg'>",
-            "fr": "Abri <img width='150px' src='./assets/layers/bike_parking/shed.svg'>",
-            "gl": "Abeiro <img width='150px' src='./assets/layers/bike_parking/shed.svg'>",
-            "de": "Schuppen <img width='150px' src='./assets/layers/bike_parking/shed.svg'>"
+            "en": "Shed <img style='width: 25%'' src='./assets/layers/bike_parking/shed.svg'>",
```

Comment:
JFYI, this could also be `w-3/12` which is also 25% https://tailwindcss.com/docs/width#fixed-width